policy/release/provenance_materials.rego

```diff
@@ -0,0 +1,79 @@
+#
+# METADATA
+# title: Provenance Materials
+# description: >-
+#   This package provides rules for verifying the contents of the materials section
+#   of the SLSA Provenance attestation.
+#
+package policy.release.provenance_materials
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.lib
+import data.lib.tkn
+
+# METADATA
+# title: Task git-clone missing
+# description: >-
+#   The attestation must contain a git-clone task with the expected commit and url results.
+# custom:
+#   short_name: missing_git_clong_task
+#   failure_msg: Task git-clone not found
+#   collections:
+#   - minimal
```

Comment:
These seem like a good candidate for the minimal collection since this is expected to pass with the most basic rhtap pipeline.